main.go

```diff
@@ -69,7 +79,11 @@ func chanHandler(feed *rss.Feed, newchannels []*rss.Channel) {
 func makeHandler(master *feeds.Feed, sourceName string) rss.ItemHandlerFunc {
 	return func(feed *rss.Feed, ch *rss.Channel, items []*rss.Item) {
 		for i := 0; i < len(items); i++ {
-			published, _ := items[i].ParsedPubDate()
+			published, err := items[i].ParsedPubDate()
+			if err != nil {
+				log.Fatal(err)
```

Comment:
The date is required in the Atom spec: https://validator.w3.org/feed/docs/atom.html#requiredFeedElements

So, I was thinking it was appropriate to exit with a `1` status, what do you think?

I also see in https://dave.cheney.net/2015/11/05/lets-talk-about-logging that he recommends bubbling up any errors to the callers, though, and then doing cleanup of files, `defer`s, etc. in `main`.